include/ccf/node/startup_config.h

```diff
@@ -64,6 +65,8 @@ struct StartupConfig : CCFConfig
 
   nlohmann::json node_data = nullptr;
 
+  RawPolicy security_policy;
```

Comment:
Should this be `std::optional` as this would only bet set on SEV-SNP?

Reply:
Yeah makes sense, wasn't sure how optional is serialised and figured an empty string would be simpler (But I don't think the current version of the code actually does this)